test/js/web/workers/worker-fixture.tsx

```diff
@@ -0,0 +1,6 @@
+postMessage("initial message");
```

Comment:
can you add actual jsx syntax into here. idk if jsx elements are transferrable due to the symbol, but they should be usable.